ksqldb-parser/src/main/java/io/confluent/ksql/util/QueryGuid.java

```diff
@@ -26,10 +26,18 @@
   private final LocalDateTime timeOfCreation;
 
   public QueryGuid(final String namespace, final String nonAnonQuery, final String anonQuery) {
+    this(namespace, nonAnonQuery, anonQuery, LocalDateTime.now());
+  }
+
+  // for testing
```

Comment:
nit: we use the `@VisibleForTesting` annotation throughout the repo for this purpose.